velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -904,20 +913,15 @@ TypePtr ExpressionFuzzer::chooseCastFromType(const TypePtr& to) {
 
 core::TypedExprPtr ExpressionFuzzer::generateCastExpression(
     const TypePtr& returnType) {
-  // Choose a random from type.
-  auto fromType = chooseCastFromType(returnType);
-  if (!fromType) {
+  const auto* callable = chooseRandomConcreteSignature(returnType, "cast");
+  if (!callable) {
     return nullptr;
   }
 
-  CallableSignature callable{"cast", {fromType}, false, returnType};
-  auto args = getArgsForCallable(callable);
+  auto args = getArgsForCallable(*callable);
 
-  // Generate try_cast expression with 50% chance.
-  bool nullOnFailure =
-      boost::random::uniform_int_distribution<uint32_t>(0, 1)(rng_);
   return std::make_shared<core::CastTypedExpr>(
-      callable.returnType, args, nullOnFailure);
+      callable->returnType, args, /*nullOnFailure*/ false);
```

Comment:
Why are we no longer randomly choosing whether nullOnFailure is true or not?

Reply:
You're right. I was confused by the recent change of converting try_cast into try(cast()), but CastTypedExpr that we're constructing here is actually one level up. I've restored the random determination of nullOnFailure back.